app/controllers/home_controller.rb

```diff
@@ -28,11 +28,12 @@ def contact_us
 
 	def send_contact_form
 		@contact_message = ContactMessage.new(message_params)
-		if @contact_message.valid? && verify_recaptcha
+		recaptcha_valid = verify_recaptcha(action: 'contact_us')
```

Comment:
I think we should stick to the 0.8 score

Reply:
But if it can be a configuration it would be better

Reply:
if not, add the score too to the registration validation

Reply:
Done